### DISCUSSION TITLE
Get the changed pixels of previous frame to current

### DISCUSSION THREAD

**Discussion by slyi** [ImageMagick/ImageMagick#6923] [Help] [answered]
### ImageMagick version

7.1.1

### Operating system, version and so on

Windows 10 64bit

### Description

Hi All,
I want to create a sequence of images, where each image contains only the changed pixels or transparent from previous frame to the current frame like the bunny sample in the [OptimizeTransparency](https://imagemagick.org/Usage/anim_opt/#opt_trans) but not on gif animation but 2 jpgs. Is the a sequence of IM commands I can use to do this?

Thanks
slyi

**Comment by snibgo** [comment, marked as answer]
Use the same command. From two input images that are the same size:
```
magick in1.png in2.png -layers OptimizeTransparency -delete 0 x.png
```
Any pixels that are the same become transparent. Changed pixels will be copied from the second input.

**Comment by slyi** [comment]
Exactly what i wanted thank you @snibgo, but it didnt work as good as i had hoped, until i added -fuzz 50% 
eg:
magick 1.jpg 2.jpg -fuzz 50%  -layers OptimizeTransparency -delete 0 diff.jpg
